block_writer/README.md

```diff
@@ -34,5 +23,5 @@ Run against an existing cockroach node or cluster.
 # Launch your node or cluster in secure mode with certificates in [mycertsdir]
 # Find a reachable address:[mycockroach:26257].
 # Run the example with:
-./block_writer https://mycockroach:26257/?certs=mycertsdir
+./block_writer "postgres://root@mycockroach:26257?sslcert=mycertsdir/root.client.crt&sslkey=mycertsdir/root.client.key"
```

Comment:
as mentioned in #39, single quotes clarify that there's no expansion

On Fri, Feb 26, 2016 at 11:59 AM, David Taylor <notifications@github.com>
wrote:

> In block_writer/README.md
> <https://github.com/cockroachdb/examples-go/pull/38#discussion_r54269763>:
>
> > @@ -34,5 +23,5 @@ Run against an existing cockroach node or cluster.
> >  # Launch your node or cluster in secure mode with certificates in [mycertsdir]
> >  # Find a reachable address:[mycockroach:26257].
> >  # Run the example with:
> > -./block_writer https://mycockroach:26257/?certs=mycertsdir
> > +./block_writer "postgres://root@mycockroach:26257?sslcert=mycertsdir/root.client.crt&sslkey=mycertsdir/root.client.key"
>
> & in the URL.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/examples-go/pull/38/files#r54269763>.
>